src/django/api/serializers/facility/facility_index_details_serializer.py

```diff
@@ -344,78 +338,3 @@ def get_created_from(self, facility):
 
     def get_has_inexact_coordinates(self, facility):
         return False
-
-    def get_sector(self, facility):
-        user_can_see_detail = can_user_see_detail(self)
-
-        use_main_created_at = is_created_at_main_date(self)
```

Comment:
Why did you remove this line, it used there (Roma this is example please ignore)

Reply:
@killev As we for now we use **sector** data both in **/facilities/** & **/facilities/<id>** endpoints and _facilities_index_details_serializer.py_ inherit from _facilities_index_serializer.py_ we don't need duplicate **getSector** method.

Reply:
@roman-stolar sorry, I demonstrated functionality of GitHub, Please ignore my comment above :)